libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -51,7 +52,7 @@ export const getFedsPlaceholderConfig = () => {
 export function getAnalyticsValue(str, index) {
   if (typeof str !== 'string' || !str.length) return str;
 
-  let analyticsValue = str.trim().replace(/[^\w]+/g, '_').replace(/^_+|_+$/g, '');
+  let analyticsValue = processTrackingLabels(str, false, 30);
```

Comment:
Just as an FYI, underscores will still be used for the AEM-based Gnav tracking that is still available on a lot of our pages. I'm hoping that analysts are aware that there will be some mismatch between the two platforms

Reply:
We've warned them and no objections were raised.  But still wouldn't be shocked if they object later.